start.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-set -o nounset
-set -o errexit
-set -o pipefail
-
-cd /var/cyhy/orchestrator || exit 1
-docker-compose up -d
```

Comment:
I know it's basic, but we should probably add a simple "Usage" section to the README for Docker Compose newbies.  For reference: https://github.com/cisagov/skeleton-docker#usage

Reply:
See commit 2a2c565.